### DISCUSSION TITLE
Generic Gauge math possible?

### DISCUSSION THREAD

**Discussion by riveter1** [nguyenquyhy/Flight-Tracker-StreamDeck#403] [Q&A]
I want to display RECIP ENG MANIFOLD PRESSURE:1 on a Generic Gauge. It displays psi but needs to be inHg. The conversion factor is 2.036. So I need to multiply the psi by 2.036 to get inHg. Is this possible?

I figured it out. Just set the units to inHg.

**Comment by nguyenquyhy** [comment]
Yup!
